<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="updateDbBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Обновить базу номенклатур</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addFilterStringBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить строку для фильтра</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Все</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Караханян</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Абраамян</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Самвел</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>10</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите ИП</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
